src/js/structs/Task.js

```diff
@@ -0,0 +1,16 @@
+import Item from './Item';
+import ServiceImages from '../constants/ServiceImages';
+
+module.exports = class Task extends Item {
+  getId() {
+    return this.get('id');
+  }
+
+  getImages() {
```

Comment:
What do you think about having specific methods for each image size instead? `getLargeImage()`, `getSmallImage()` etc